### PR TITLE
ApiCheck: Update baselines to 2.1

### PR DESCRIPTION
**Pull request by bricelam**
Better late than never...

### CODE REVIEW

File: src/EFCore.Relational/breakingchanges.netcore.json [hunk 1]
```diff
@@ -1,119 +1,34 @@
   [
     {
-      "TypeId": "public sealed class Microsoft.EntityFrameworkCore.Migrations.MigrationsSqlGeneratorDependencies",
-      "MemberId": "public .ctor(Microsoft.EntityFrameworkCore.Storage.IRelationalCommandBuilderFactory commandBuilderFactory, Microsoft.EntityFrameworkCore.Storage.ISqlGenerationHelper sqlGenerationHelper, Microsoft.EntityFrameworkCore.Storage.IRelationalTypeMapper typeMapper)",
+      "TypeId": "public class Microsoft.EntityFrameworkCore.Query.Sql.DefaultQuerySqlGenerator : Remotion.Linq.Parsing.ThrowingExpressionVisitor, Microsoft.EntityFrameworkCore.Query.Sql.ISqlExpressionVisitor, Microsoft.EntityFrameworkCore.Query.Sql.IQuerySqlGenerator",
+      "MemberId": "protected virtual System.Void GenerateList(System.Collections.Generic.IReadOnlyList<System.Linq.Expressions.Expression> items, System.Action<Microsoft.EntityFrameworkCore.Storage.IRelationalCommandBuilder> joinAction = null)",
```
Comment (bricelam) [inline src/EFCore.Relational/breakingchanges.netcore.json]: Unintentional

File: src/EFCore.Relational/breakingchanges.netcore.json [hunk 2]
```diff
@@ -1,119 +1,34 @@
...
       "Kind": "Removal"
     },
     {
-      "TypeId": "public sealed class Microsoft.EntityFrameworkCore.Update.UpdateSqlGeneratorDependencies",
-      "MemberId": "public .ctor(Microsoft.EntityFrameworkCore.Storage.ISqlGenerationHelper sqlGenerationHelper)",
+      "TypeId": "public class Microsoft.EntityFrameworkCore.Query.Sql.DefaultQuerySqlGenerator : Remotion.Linq.Parsing.ThrowingExpressionVisitor, Microsoft.EntityFrameworkCore.Query.Sql.ISqlExpressionVisitor, Microsoft.EntityFrameworkCore.Query.Sql.IQuerySqlGenerator",
+      "MemberId": "protected virtual System.Void GenerateList<T0>(System.Collections.Generic.IReadOnlyList<T0> items, System.Action<T0> generationAction, System.Action<Microsoft.EntityFrameworkCore.Storage.IRelationalCommandBuilder> joinAction = null)",
```
Comment (bricelam) [inline src/EFCore.Relational/breakingchanges.netcore.json]: Unintentional

File: src/EFCore/breakingchanges.netcore.json [hunk 3]
```diff
@@ -1,117 +1,45 @@
...
-  {
-    "TypeId": "public interface Microsoft.EntityFrameworkCore.Metadata.IEntityType : Microsoft.EntityFrameworkCore.Metadata.ITypeBase",
-    "MemberId": "System.Collections.Generic.IEnumerable<Microsoft.EntityFrameworkCore.Metadata.IServiceProperty> GetServiceProperties()",
-    "Kind": "Addition"
-  }
-]
+  [
+    {
+      "TypeId": "public class Microsoft.EntityFrameworkCore.Storage.ValueConversion.CharToStringConverter : Microsoft.EntityFrameworkCore.Storage.ValueConversion.ValueConverter<System.Char, System.String>",
+      "Kind": "Removal"
```
Comment (bricelam) [inline src/EFCore/breakingchanges.netcore.json]: Where did they go?

File: src/EFCore/breakingchanges.netcore.json [hunk 4]
```diff
@@ -1,117 +1,45 @@
...
+    },
+    {
+      "TypeId": "public class Microsoft.EntityFrameworkCore.Storage.ValueConversion.TimeSpanToStringConverter : Microsoft.EntityFrameworkCore.Storage.ValueConversion.ValueConverter<System.TimeSpan, System.String>",
+      "Kind": "Removal"
+    },
+    {
+      "TypeId": "public abstract class Microsoft.EntityFrameworkCore.Query.EntityQueryModelVisitor : Remotion.Linq.QueryModelVisitorBase",
+      "MemberId": "protected static System.Reflection.MethodInfo get_SelectAsyncMethod()",
+      "Kind": "Removal"
```
Comment (bricelam) [inline src/EFCore/breakingchanges.netcore.json]: Could this break providers?

File: src/EFCore/breakingchanges.netcore.json (hunk 3 above)
Comment (ajcvickers) [reply src/EFCore/breakingchanges.netcore.json]: Didn't we introduce a new base type here?

File: src/EFCore/breakingchanges.netcore.json (hunk 4 above)
Comment (AndriySvyryd) [reply src/EFCore/breakingchanges.netcore.json]: Unlikely for relation providers

File: src/EFCore.Relational/breakingchanges.netcore.json (hunk 1 above)
Comment (bricelam) [reply src/EFCore.Relational/breakingchanges.netcore.json]: False positive.

File: src/EFCore.Relational/breakingchanges.netcore.json (hunk 2 above)
Comment (bricelam) [reply src/EFCore.Relational/breakingchanges.netcore.json]: False positive.

**Pull request merged**